client/pages/server/login.vue

```diff
@@ -57,23 +57,6 @@ export default Vue.extend({
   layout: 'fullpage',
   middleware: 'serverMiddleware',
   auth: false,
-  async asyncData({ store, redirect, $api }) {
-    try {
-      await store.dispatch(
-        'servers/connectServer',
-        store.state.servers.serverUsed.address
-      );
-
-      const publicUsers = (await $api.user.getPublicUsers({})).data;
-
-      const brandingData = (await $api.branding.getBrandingOptions()).data;
```

Comment:
As @ThibaultNocchi mentioned, moving the branding to `fetch()` makes it a bit jarring (Same for public users).

As a proper fix for them not being refreshed, while keeping the nicer experience, we can add an exclude list to the `keep-alive` here: https://github.com/jellyfin/jellyfin-vue/blob/master/client/layouts/fullpage.vue#L4

Just exclude the server pages (We might use fullpage for other things in the future) and that should fix these not being re-fetched when in `asyncData`.

Reply:
More info on the available keep-alive props: https://v3.vuejs.org/api/built-in-components.html#keep-alive